python/lsst/ts/xml/generate_subsystems_doc.py

```diff
@@ -294,9 +294,13 @@ def generate_subsystems_doc():
                         state_name = states[0].split("_")[0].lstrip()
                         cf.write(f":{state_name}:\n")
                         for state in states:
-                            cf.write(
-                                f"  * :any:`{state.split('_')[1]} <lsst.ts.idl.enums.{subsystem}.{state_name}.{state.split('_')[1].upper()}>`\n"  # noqa
-                            )
+                            # state format starts as [\n statename_state ]
```

Comment:
At line 294 please use `state_name = states[0].split("_", 1)[0].lstrip()`

In other words: add the `1` argument to `split`. There is no point to splitting past the first "_".

Reply:
Thanks for documenting the format. However, itcan include a value. Thus please consider something along the lines of:
```
# enum item format is `\n enumClassName_itemName [= value]`
# where itemName may include underscores,
# and all whitespace is optional and of arbitrary length.
```

Reply:
Please consider using `enum_class_name` instead of `state_name` to avoid confusion. Each enum entry typically does describe a single state (though there may be exceptions). But even if so, enum class name is *never* a state.

I'm sure there are other terms. But C++ and Python both use the term "enum class", and Java uses "enum type", which is close. So "enum_class_name" is a decent choice.